### DISCUSSION TITLE
Some VsCode issue

### DISCUSSION THREAD

**Discussion by Aashim-Limbu** [Cyfrin/foundry-full-course-cu#2863] [Q&A]
**In Patrick's video, there is a suggestion:**

![image](https://github.com/user-attachments/assets/329cc4a1-13ce-4fe3-a83f-44abb692188f)

**However, in my VSCode, the suggestion doesn't appear. I wonder why:**

![image](https://github.com/user-attachments/assets/7291cf37-57fb-4ae9-9ffc-1319a2e787ec)

**I have ensured that I have the extension and `settings.json` configured properly. Am I the only one encountering this issue/bug in VSCode?**

![image](https://github.com/user-attachments/assets/1b5ae9fe-f58e-4edc-a2bf-436e2c03a2f0)

**Comment by EngrPips** [comment]
Hello @Aashim-Limbu, Have you tried closing your `vscode` and relaunching it?

**Reply by Aashim-Limbu** [reply]
Yes, I have restarted my VS Code. The issue is with the suggestions — they are not showing up there. It's quite a bad experience without them So, I raised this poll to see if I missed any extensions.

@EngrPips, thank you for your response! 👍

**Reply by EngrPips** [reply]
I show the extensions I have installed in the below image. 

<img width="1440" alt="Screenshot 2024-11-14 at 10 25 07" src="https://github.com/user-attachments/assets/cc70a486-e680-4ddb-a981-e7faab23c7be">

Also the suggestion seems to work for me as well.

**Reply by LordAdhaar** [reply]
Is it just with makeAddr or no suggestions are working?

**Reply by EngrPips** [reply]
I assume no suggestions are working.

**Reply by HonourJAH** [reply]
I think you should keep doing the typing bro. It will work eventually. I experienced the same issue for about 2 wks before everything started working fine.

**Reply by Aashim-Limbu** [reply]
### Issue with Suggestions in VS Code

Yeah, the error persists only for `makeAddress`.  
![image](https://github.com/user-attachments/assets/0883ef86-c82d-47a8-a720-ac5cffee2c78)

The suggestion is there for the instance that I import (e.g., `fundme` in this case).  
![image](https://github.com/user-attachments/assets/c990b850-45f6-488a-9057-282cdeeacae4)

The problem is only with external library-specific items like `makeAddr`, and it doesn't recognize `vm` inside the test as well it is quite strange that it is working in someone else machine but not in mine.   
![image](https://github.com/user-attachments/assets/6649ae75-c951-4d49-bd17-897067499591)

**Reply by EngrPips** [reply]
That is kind of weird. I haven't experienced this, so it is kind of hard to tell what could be wrong. But that shouldn't stop you from progressing with the course and I hope it eventually works as expected later on

**Reply by Aashim-Limbu** [reply]
### Switched to Juan Blanco version and now working as expected

![image](https://github.com/user-attachments/assets/4639e6e5-8a9c-4499-8b4e-b90dce826170)

Everyone, thank you for the response. 🙏

**Reply by EngrPips** [reply]
Interesting. I am glad you figured it out.
